worker.rb

```diff
@@ -44,7 +44,8 @@ def create_vm(id, name, cpus, memory, volume_capacity, ip, config)
                 puts "Found new pending VM!"
                 puts "Name: #{row["name"]}"
                 mac = create_vm(row["id"], row["name"], row["cpus"], row["memory"], row["volume_capacity"], row["ip"], config)
-                conn.exec("UPDATE #{config["table"]} SET mac=\"#{mac}\" WHERE id=#{row["id"]}")
+                conn.exec("UPDATE #{config["table"]} SET mac=\"#{mac}\" AND status=null WHERE id=#{row["id"]}")
```

Comment:
This isn't the correct way to update multiple columns at the same time. Syntax:

`UPDATE table_name SET expression = value, expression = value WHERE clause`